tslint-sonarts.json

```diff
@@ -15,6 +15,7 @@
     "no-misspelled-operator": true,
     "no-inconsistent-return": false,
     "no-unconditional-jump": true,
-    "no-misleading-array-reverse": true
+    "no-misleading-array-reverse": true,
+    "no-dead-stores": true
```

Comment:
:+1: 
I had thought the same (and indeed the test files were still singular) but then something convinced me of the plural name. I don't remember what anymore, so switched back to singular as you suggest.